deposit_contract/contracts/validator_registration.v.py

```diff
@@ -21,6 +21,21 @@ def __init__():
         self.zerohashes[i+1] = sha3(concat(self.zerohashes[i], self.zerohashes[i]))
         self.branch[i+1] = self.zerohashes[i+1]
 
+@public
+def to_little_endian_64(value: uint256) -> bytes[8]:
```

Comment:
Maybe add a `@constant` decorator?

Reply:
Done. Also restricted usage of `to_little_endian_64` to values `<= 2**64 -1`